modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/styles/containers/_dxp-activation-keys-table.scss

```diff
@@ -0,0 +1,27 @@
+.cp-dxp-activation-key-container {
+	max-width: 1271px;
+}
+
+.cp-dxp-activation-key-table {
+	max-width: 85vw;
+
+	tbody {
+		tr:first-child {
+			td {
+				border-radius: 0 !important;
+				border-top: 1px solid #e1e1e4 !important;
```

Comment:
Could be exchange to class, border-radius and color? 
`border-0 border-top-0 border-right-0 border-bottom-0 border-left-0`

Reply:
Sure, @Lucas2691! Actually, I saw that the `border-radius: 0 !important;` is not needed, so I'll be removing. Thanks!